titiler/cmr/backend.py

```diff
@@ -37,7 +37,7 @@
 
 @cached(  # type: ignore
     TTLCache(maxsize=100, ttl=60),
-    key=lambda auth, daac: hashkey(auth.tokens[0]["access_token"], daac),
+    key=lambda auth, daac: hashkey(auth.token["access_token"], daac),
```

Comment:
A fix to update the API following https://github.com/nsidc/earthaccess/pull/782, however we are using `iam`, as an `S3_AUTH_STRATEGY`, not edl, so when deployed without the IAM auth strategy the API will break because no earthaccess username and password is configured to get a token.